packages/core/src/api/events/initialize.ts

```diff
@@ -0,0 +1,37 @@
+import type { Config } from '../../config';
+import type { InternalLogger } from '../../internalLogger';
+import type { Metas } from '../../metas';
+import { TransportItem, TransportItemType, Transports } from '../../transports';
```

Comment:
Those are all types if i understand correctly, hence:
```suggestion
import type { TransportItem, TransportItemType, Transports } from '../../transports';
```

Reply:
?!

Reply:
`TransportItemType` is used as a value :/